app/components/user-profile-permissions.js

```diff
@@ -31,99 +32,84 @@ export default class UserProfilePermissionsComponent extends Component {
   @tracked instructedSessions = [];
   @tracked studentAdvisedSessions = [];
 
+  get courseCount() {
+    return (
+      this.directedCourses.length +
+      this.administeredCourses.length +
+      this.instructedCourses.length +
+      this.studentAdvisedCourses.length
+    );
+  }
+
+  get sessionCount() {
+    return (
+      this.administeredSessions.length +
+      this.instructedSessions.length +
+      this.studentAdvisedSessions.length
+    );
+  }
+
   @restartableTask
   *load() {
     this.academicYearCrossesCalendarYearBoundaries = yield this.iliosConfig.itemFromConfig(
       'academicYearCrossesCalendarYearBoundaries'
     );
 
-    const map = yield hash({
-      schools: this.store.findAll('school'),
-      academicYears: this.store.findAll('academic-year'),
-      defaultSchool: this.args.user.school,
-    });
-
-    this.schools = map.schools;
-    this.academicYears = map.academicYears;
+    this.schools = (yield this.store.findAll('school', { reload: true })).toArray();
+    this.academicYears = (yield this.store.findAll('academic-year', { reload: true })).toArray();
```

Comment:
Why the reload?

Reply:
> Is there any way to add stickiness to the expanded state (or not) of Programs, Program Years, Courses, and Sessions so that when the user comes back to the Admin Console, those values would be in the same state as before the user followed the link?

technically, that's all doable. the question is - how far do you want to spin that thread... i can't answer that. if this is important, then pls file a follow up ticket and i'll get to it asap.

Reply:
you're right, this is not necessary here. i went overboard when dealing with the incomplete schools loading behaviour. i now removed that flag :heavy_check_mark: